django_celery_extensions/task.py

```diff
@@ -168,6 +169,15 @@ class DjangoTask(Task):
 
     ignore_task_after_success_timedelta = None
 
+    def __new__(cls, *args, **kwargs):
```

Comment:
unfortunatelly set default arguments from queue is not as simple as may seen. Because there is no method in Celer which creates Task class (there is one which generates task class from function, but If you want to define celery task with class it will not work). So the best solution is override __new__ class of the celery base task and set default arguments here